CDP4SiteDirectory/Views/Dialogs/PersonDialog.xaml

```diff
@@ -19,7 +20,7 @@
     <dx:DXWindow.Resources>
         <ResourceDictionary>
             <cdp4Composition:ThingToIconUriConverter x:Key="ThingToIconUriConverter" />
-            <BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter" />
+            <cdp4Converters:BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter" />
```

Comment:
This used the standard MS converter, not the one stored in cdp4. Had to change it in order to inverse Boolean.